feeds/wlan-ap/opensync/src/platform/openwrt/src/lib/target/src/stats.c

```diff
@@ -148,6 +148,14 @@ bool target_stats_clients_get(radio_entry_t *radio_cfg, radio_essid_t *essid,
         return ret;
 }
 
+uint64_t update_stats_feild(uint64_t new_value, uint64_t old_value) {
```

Comment:
field not feild please.

Needs to be all or none.  If one count has been reset, they've all been reset.  Therefore test all and then either subtract all or not.

In stat_nl80211.c/nl80211_assoclist_recv(), add NL80211_STA_INFO_CONNECTED_TIME to the stats_policy.  Then add it to the stored data target_client_record_t and use it to make the decision.

It doesn't have to be added to the app_client_record_t (would be a mess and require a Cloud change as well).